app/helpers/view_components/mentor/header.rb

```diff
@@ -101,8 +101,8 @@ def automation_tab
           class: tab_class(:automation)
         ) do
           graphical_icon(:automation) +
-            tag.span("Automation") +
-            tag.span(number_with_delimiter(num_representations_without_feedback), class: 'count')
+            tag.span("Automation") #+
+          # tag.span(number_with_delimiter(num_representations_without_feedback), class: 'count')
```

Comment:
This count currently takes ages. Caching the individual track counts and summing them here would be the best bet. And we can then reuse those caches for the track counts as well.